app/src/main/java/cm/aptoide/pt/home/more/eskills/EskillsInfoFragment.kt

```diff
@@ -0,0 +1,262 @@
+    val faq3 = faqsView.findViewById<View>(R.id.eskills_faqs_item_3)
+
+    setupFAQItem(faq1, R.string.eskills_v2_faqs_1_title, R.string.eskills_v2_faqs_1_body_1)
+    setupFAQItem(faq2, R.string.eskills_v2_faqs_2_title, R.string.eskills_v2_faqs_2_body_1)
+    setupFAQItem(faq3, R.string.eskills_v2_faqs_3_title, R.string.eskills_v2_faqs_3_body_1)
+  }
+
+  private fun setupFAQItem(faq: View, question: Int, answer: Int) {
+    faq.findViewById<TextView>(R.id.eskills_faq_question).setText(question)
+    val answerView = faq.findViewById<TextView>(R.id.eskills_faq_answer)
+    val arrow = faq.findViewById<ImageView>(R.id.eskills_faq_arrow)
+    answerView.setText(answer)
+
+    faq.setOnClickListener {
+      if (answerView.visibility == View.VISIBLE) {
+        answerView.visibility = View.GONE
+        arrow.setImageDrawable(ContextCompat.getDrawable(context!!, R.drawable.ic_down_arrow))
+      } else {
+        answerView.visibility = View.VISIBLE
+        arrow.setImageDrawable(ContextCompat.getDrawable(context!!, R.drawable.ic_up_arrow))
+      }
+    }
+  }
+
+  override fun handleMoreAppsClick(): Observable<Void> {
+    return RxView.clicks(seeAllButton)
+  }
+
+  override fun handleLearnMoreClick(): Observable<Void> {
+    return RxView.clicks(learnMoreCard)
+  }
+
+  override fun handleWalletDisclaimerClick(): Observable<Void> {
+    return RxView.clicks(walletDisclaimer)
+  }
+
+  override fun scrollToInfo() {
+    val scrollY = learnMoreSection.top - 32
+    mainScrollView.smoothScrollTo(0, scrollY)
+  }
+
+  override fun getHistoryTracker(): ScreenTagHistory? {
+    return ScreenTagHistory.Builder.build(
+      this.javaClass.simpleName
+    )
+  }
+
+  override fun onCreateView(
+    inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?
+  ): View? {
+    return inflater.inflate(R.layout.fragment_eskills_info, container, false)
+  }
+
+  // Recycler view
+  override fun setupHeaderListener() = Unit
+
+  override fun showResultsVisibility() {
+    apps_list.visibility = View.VISIBLE
+    error_view.visibility = View.GONE
+    progress_bar.visibility = View.GONE
+  }
+
+  override fun showErrorVisibility() {
+    error_view.visibility = View.VISIBLE
+    apps_list.visibility = View.GONE
+    progress_bar.visibility = View.GONE
+  }
+
+  override fun getItemSizeWidth(): Int {
+    return 104
+  }
+
+  override fun getItemSizeHeight(): Int {
+    return 165
+  }
+
+  override fun getAdapterStrategy(): GridRecyclerView.AdaptStrategy {
+    return GridRecyclerView.AdaptStrategy.SCALE_KEEP_ASPECT_RATIO
+  }
+
+  override fun createViewHolder(): (ViewGroup, Int) -> ListAppsEskillsViewHolder {
+    return { parent, _ ->
+      ListAppsEskillsViewHolder(
+        LayoutInflater.from(parent.context).inflate(
+          R.layout.eskills_app_home_item, parent, false
+        ), DecimalFormat("0.0")
+      )
+    }
+  }
+
+  override fun refreshEvents(): Observable<Void> {
+    return Observable.empty()
```

Comment:
i assumed this is because you didnt need it and it was on the interface, but just to confirm ..

Reply:
yes, but it wasn't needed. I removed the overriding method